Formula/anime-downloader.rb

```diff
@@ -1,10 +1,10 @@
 class AnimeDownloader < Formula
   include Language::Python::Virtualenv
 
-  desc "Simple but powerful anime downloader and streamer"
+  desc "Download your favourite anime"
```

Comment:
Not quite sure why we need to update the `desc` but the old desc seems good.

Reply:
Ah, just updated it to ensure consistency with the description on PyPI.

Reply:
Sounds cool!